docs/ADRs/012-tad-tool.md

```diff
@@ -0,0 +1,50 @@
+# ADR-0012 Tool for Transparency of Algorithmic Decision making
+
+## Context
+
+We are considering tooling for organisations to get more grip on their algorithms. Tooling for, for instance bias and fairness tests, and assessments (like [IAMA](https://www.rijksoverheid.nl/documenten/rapporten/2021/02/25/impact-assessment-mensenrechten-en-algoritmes)).
+
+Transparency, we think, can be fostered by sharing reports from such a tool in a standardized way.
+
+There are several existing open source
+[tools](https://minbzk.github.io/ai-validation/Projects/TAD/Existing%20Tools/Comparison/tools/) which
+we [have assessed](https://minbzk.github.io/ai-validation/Projects/TAD/Existing%20Tools/Comparison/tools_comparison/).
+Some support only assessments, others already combine more features and can generate a report.
+There is however no tool
+that supports all the [requirements](https://minbzk.github.io/ai-validation/Projects/TAD/Existing Tools/Comparison/requirements/) we have.
```

Comment:
```suggestion
that supports all the [requirements](../Projects/TAD/Existing Tools/Comparison/requirements/) we have.
```

Reply:
Oh great, it resolves.. check, will remember for next time.